### ISSUE TITLE
Wan2.2 i2v-A14B LoRa training has no effect. Comparison with other trainers AI-Toolkit

### ISSUE THREAD

**Issue by wzgrx** [kohya-ss/musubi-tuner#621]
Device: Windows 11 4090 24GB 96GB RAM
Video material, resolution below 960, 16fps, duration 5s

Training 2D motion effects with the same material and settings, using AI Toolkit, LoRa works fine, but Musubi Tuner doesn't.Need help from experts and professionals to successfully implement musubi-tuner training lora and achieve the effect of using AI Toolkit. The following includes all the settings

video dataests
https://huggingface.co/datasets/wzgrx/2D-animation-effects

using AI Toolkit trian lora
https://civitai.com/models/1920897/wan22-2d-animation-effects-2d
Test results
https://github.com/user-attachments/assets/37332aa7-6286-49ef-9241-157bb82b7580

using musubi-tuner

same video

[general]
caption_extension = ".txt"
batch_size = 1
enable_bucket = false
[[datasets]]
image_directory = "E:/AI/musubi-tuner/train/image"
cache_directory = "E:/AI/musubi-tuner/train/cache_image"
resolution = [1024, 1024]  
num_repeats = 1
[[datasets]]
video_directory = "E:/AI/musubi-tuner/train/video"
cache_directory = "E:/AI/musubi-tuner/train/cache_video"
frame_extraction = "uniform"
source_fps = 16.0
target_frames = [57]
max_frames = 57
enable_bucket = true
bucket_no_upscale = false
resolution = [384, 384]

python src/musubi_tuner/wan_cache_latents.py `
  --dataset_config "E:\AI\musubi-tuner\train\2.toml" `
  --vae "E:\AI\musubi-tuner\wan\wan_2.1_vae.safetensors" `
  --vae_cache_cpu `
  --batch_size 2 `
  --i2v

python src/musubi_tuner/wan_cache_text_encoder_outputs.py `
  --dataset_config "E:\AI\musubi-tuner\train\2.toml" `
  --t5 "E:\AI\musubi-tuner\wan\models_t5_umt5-xxl-enc-bf16.pth" `
  --fp8_t5 `
  --batch_size 4

accelerate launch --num_cpu_threads_per_process 1 src\musubi_tuner\wan_train_network.py `
  --task i2v-A14B `
  --dataset_config "E:\AI\musubi-tuner\train\2.toml" `
  --dit "E:\AI\musubi-tuner\wan\wan2.2_i2v_high_noise_14B_fp16.safetensors" `
  --vae "E:\AI\musubi-tuner\wan\wan_2.1_vae.safetensors" `
  --t5 "E:\AI\musubi-tuner\wan\models_t5_umt5-xxl-enc-bf16.pth" `
  --flash_attn --split_attn `
  --mixed_precision fp16 `
  --fp8_base --fp8_scaled `
  --optimizer_type adamw8bit --learning_rate 2e-4 `
  --gradient_checkpointing `
  --max_data_loader_n_workers 8 --persistent_data_loader_workers `
  --network_module networks.lora_wan --network_dim 16 --network_alpha 16 `
  --timestep_sampling shift --discrete_flow_shift 5.0 `
  --min_timestep 900 --max_timestep 1000 `
  --preserve_distribution_shape `
  --max_train_epochs 20 --save_every_n_epochs 2 --seed 42 `
  --output_dir "E:\AI\musubi-tuner\out" --output_name "my_i2v_lora_high" `
  --log_with tensorboard `
  --logging_dir "E:\AI\musubi-tuner\train\log" `
  --blocks_to_swap 16 `
  --offload_inactive_dit 

accelerate launch --num_cpu_threads_per_process 1 --mixed_precision fp16 `
  .\src\musubi_tuner\wan_train_network.py `
  --task i2v-A14B `
  --dataset_config "E:\AI\musubi-tuner\train\2.toml" `
  --dit "E:\AI\musubi-tuner\wan\wan2.2_i2v_low_noise_14B_fp16.safetensors" `
  --xformers `
  --vae "E:\AI\musubi-tuner\wan\wan_2.1_vae.safetensors" `
  --t5 "E:\AI\musubi-tuner\wan\models_t5_umt5-xxl-enc-bf16.pth" `
  --fp8_base --fp8_scaled `
  --optimizer_type adamw8bit --learning_rate 2e-5 `
  --gradient_checkpointing `
  --max_data_loader_n_workers 8 --persistent_data_loader_workers `
  --network_module networks.lora_wan --network_dim 16 --network_alpha 16 `
  --network_args "loraplus_lr_ratio=4" `
  --timestep_sampling shift --discrete_flow_shift 5.0 `
  --min_timestep 0 --max_timestep 900 `
  --preserve_distribution_shape `
  --max_train_epochs 20 --save_every_n_epochs 2 --seed 42 `
  --output_dir "E:\AI\musubi-tuner\out" --output_name "my_i2v_lora_low" `
  --blocks_to_swap 18 `
  --log_with tensorboard `
  --logging_dir "E:\AI\musubi-tuner\train\log" `
  --offload_inactive_dit 

After a period of training, the generated lora is completely different from the lora generated using AI Toolkit. The lora trained with musubi-tuner has no effect at all. What should I modify? Thank you very much

**Comment by kohya-ss** [comment]
It looks like you are training a high noise model with a timestep range of 0-900. 1000 is close to noise, so `--min_timestep 900 --max_timestep 1000` is probably what you intended.

**Comment by wzgrx** [comment]
> It looks like you are training a high noise model with a timestep range of 0-900. 1000 is close to noise, so `--min_timestep 900 --max_timestep 1000` is probably what you intended.

Yes, but can you train a lora with this effect using the same materials? It seems that there is nothing wrong with the command.

**Comment by kohya-ss** [comment]
The standard setting for inference is to use 900-1000 timesteps for the high noise model and 0-900 timesteps for the low noise model. The LoRA is not trained with the 900-1000 timesteps with the initial setting, which is probably why LoRA is ineffective.

**Comment by wzgrx** [comment]
> The standard setting for inference is to use 900-1000 timesteps for the high noise model and 0-900 timesteps for the low noise model. The LoRA is not trained with the 900-1000 timesteps with the initial setting, which is probably why LoRA is ineffective.

Obviously, my settings are correct, the directory is also hight 900-100, low0-900, but the training is still invalid. You can try it yourself. The material is connected above

**Comment by kohya-ss** [comment]
Sorry, I overlooked. Your setup doesn't seem to have any major issues.

I don't think `--fp8_t5` is necessary when caching the Text Encoder outputs. However, it probably won't have such a big impact that training won't progress at all.

Could you share your AI-Toolkit settings?

**Comment by wzgrx** [comment]
> Sorry, I overlooked. Your setup doesn't seem to have any major issues.
> 
> I don't think is necessary when caching the Text Encoder outputs. However, it probably won't have such a big impact that training won't progress at all.`--fp8_t5`
> 
> Could you share your AI-Toolkit settings?

---
job: "extension"
config:
  name: "13"
  process:
    - type: "diffusion_trainer"
      training_folder: "E:\\AI-Toolkit-Easy-Install\\AI-Toolkit-Easy-Install\\AI-Toolkit\\output"
      sqlite_db_path: "./aitk_db.db"
      device: "cuda"
      trigger_word: null
      performance_log_every: 10
      network:
        type: "lora"
        linear: 32
        linear_alpha: 32
        conv: 16
        conv_alpha: 16
        lokr_full_rank: true
        lokr_factor: -1
        network_kwargs:
          ignore_if_contains: []
      save:
        dtype: "bf16"
        save_every: 250
        max_step_saves_to_keep: 4
        save_format: "diffusers"
        push_to_hub: false
      datasets:
        - folder_path: "E:\\AI-Toolkit-Easy-Install\\AI-Toolkit-Easy-Install\\AI-Toolkit\\datasets/2d"
          mask_path: null
          mask_min_value: 0.1
          default_caption: ""
          caption_ext: "txt"
          caption_dropout_rate: 0.05
          cache_latents_to_disk: false
          is_reg: false
          network_weight: 1
          resolution:
            - 240
          controls: []
          shrink_video_to_frames: true
          num_frames: 33
          do_i2v: true
          flip_x: false
          flip_y: false
      train:
        batch_size: 1
        bypass_guidance_embedding: false
        steps: 3000
        gradient_accumulation: 1
        train_unet: true
        train_text_encoder: false
        gradient_checkpointing: true
        noise_scheduler: "flowmatch"
        optimizer: "adamw8bit"
        timestep_type: "shift"
        content_or_style: "content"
        optimizer_params:
          weight_decay: 0.0001
        unload_text_encoder: false
        cache_text_embeddings: true
        lr: 0.0001
        ema_config:
          use_ema: false
          ema_decay: 0.99
        skip_first_sample: true
        force_first_sample: false
        disable_sampling: true
        dtype: "bf16"
        diff_output_preservation: false
        diff_output_preservation_multiplier: 1
        diff_output_preservation_class: "person"
        switch_boundary_every: 20
        loss_type: "mse"
      model:
        name_or_path: "ai-toolkit/Wan2.2-I2V-A14B-Diffusers-bf16"
        quantize: true
        qtype: "qfloat8"
        quantize_te: true
        qtype_te: "qfloat8"
        arch: "wan22_14b_i2v"
        low_vram: true
        model_kwargs:
          train_high_noise: true
          train_low_noise: false
      sample:
        sampler: "flowmatch"
        sample_every: 250
        width: 1024
        height: 1024
        samples: []
        neg: ""
        seed: 42
        walk_seed: true
        guidance_scale: 4
        sample_steps: 25
        num_frames: 41
        fps: 16
meta:
  name: "[name]"
  version: "1.0"

**Comment by kohya-ss** [comment]
It seems that the AI-Toolkit settings only train the high noise model. Other settings such as fp16/bf16, resolution, number of frames, rank/alpha, number of steps, and learning rate are significantly different, so it is possible that the training results will differ.

If you could tell us why you are changing the settings from AI-Toolkit, we may be able to give you advice on settings that reflect your intentions.

**Comment by AIMixer** [comment]
@wzgrx 
Hello, I used your dataset to train this LoRA model based on musubi-tuner, and the results are excellent! The main difference in my approach is that I specified both high/low noise and adjusted the timestep settings.

Here's a demo video I recorded showing the effects: [bilibili.com/video/BV1jwxazaENB/](bilibili.com/video/BV1jwxazaENB/)

For your reference, LoRAMaster is a GUI toolkit built around musubi. Project [https://github.com/AIMixer/LoRAMaster](https://github.com/AIMixer/LoRAMaster)

A big thanks to kohya-ss for developing the fantastic musubi-tuner project!

**Comment by TTTTong777** [comment]
> 你好，我用你的数据集训练了基于musubi-tuner的LoRA模型，结果非常出色！我方法的主要区别是我同时指定了高低噪声，并调整了时间步长设置。
> 
> 这是我录制的演示视频，展示了这些效果：[bilibili.com/video/BV1jwxazaENB/](bilibili.com/video/BV1jwxazaENB/)
> 
> 供你参考，LoRAMaster 是一个基于 musubi 构建的图形用户界面工具包。https://github.com/AIMixer/LoRAMaster 项目
> 
> 非常感谢kohya-ss开发了这个精彩的musubi调谐器项目！

同事指定高低噪声的情况下，时间步长该如何分配呢
